data/dynamic_wallpaper.gresource.xml

```diff
@@ -3,6 +3,6 @@
   <gresource prefix="/me/dusansimic/DynamicWallpaper">
```

Comment:
Now that I've noticed it, should this file be named with the app id (`me.dusansimic.DynamicWallpaper`) instead of `dynamic_wallpaper` like the rest of files?